README.md

```diff
@@ -1,67 +1,56 @@
-Certain features are only available with NixOS.
+Unlike color scheme providers such as
+[base16.nix](https://github.com/SenchoPens/base16.nix) or
+[nix-colors](https://github.com/Misterio77/nix-colors), Stylix goes further by
+applying themes to supported applications, following the "it just works"
+philosophy.
 
 ## Resources
 
-Please refer to the [Stylix book](https://danth.github.io/stylix/)
-for instructions and a list of supported apps.
-
-For a visual guide, watch the [*Ricing Linux Has Never Been Easier | NixOS +
-Stylix*](https://youtu.be/ljHkWgBaQWU) YouTube video by
-[Vimjoyer](https://www.youtube.com/@vimjoyer).
-
-> [!NOTE]
->
-> It's now necessary to include `stylix.enable = true` in your configuration
-> for any other settings to take effect. This is not mentioned in the video
-> linked above.
-
-If you have any questions, you are welcome to
-join our [Matrix room](https://matrix.to/#/#stylix:danth.me),
-or ask on [GitHub Discussions](https://github.com/danth/stylix/discussions).
+- [Documentation](https://danth.github.io/stylix)
+- [Matrix room](https://matrix.to/#/#stylix:danth.me)
 
 ## Example configurations
 
 ### GNOME 46
 
-![GNOME 46](./gnome.png)
+<figure>
+    <img src="./gnome.png" alt="GNOME 46" width=100%>
+    <figcaption>
+        Wallpapers by <a
+        href="https://unsplash.com/photos/three-bicycles-parked-in-front-of-building-hwLAI5lRhdM">
+        Clay Banks</a> and <a
+        href="https://unsplash.com/photos/brown-road-in-forest-during-daytime-L505cPnmIds">
+        Derrick Cooper</a>.
+    </figcaption>
+</figure>
 
-Photos by [Clay Banks](https://unsplash.com/photos/three-bicycles-parked-in-front-of-building-hwLAI5lRhdM)
-and [Derrick Cooper](https://unsplash.com/photos/brown-road-in-forest-during-daytime-L505cPnmIds).
+Try a live demo of this dark theme by running:
 
-Try a live demo of this theme by running
-`nix run github:danth/stylix#testbed:gnome:default:dark:image:scheme`.
+```console
+nix run github:danth/stylix#testbed:gnome:default:dark:image:scheme
+```
 
 ### KDE Plasma 5
 
-![KDE Plasma 5](./kde.png)
-
-Photos by [Aniket Deole](https://unsplash.com/photos/mountain-surrounded-by-trees-under-cloudy-sky-T-tOgjWZ0fQ)
-and [Tom Gainor](https://unsplash.com/photos/landscape-photography-of-body-of-water-overlooking-mountain-range-ZqLeQDjY6fY).
-
-KDE theming is still a work in progress - so some manual steps may be needed
-to apply the settings completely.
+<figure>
+    <img src="./kde.png" alt="KDE Plasma 5" width=100%>
+    <figcaption>
+        Wallpapers by <a
+        href="https://unsplash.com/photos/mountain-surrounded-by-trees-under-cloudy-sky-T-tOgjWZ0fQ">
+        Aniket Deole</a> and <a
+        href="https://unsplash.com/photos/landscape-photography-of-body-of-water-overlooking-mountain-range-ZqLeQDjY6fY">
+        Tom Gainor</a>.
+    </figcaption>
+</figure>
+
+Since KDE theming is still WIP, some manual steps may be required to properly
+apply its theme.
```

Comment:
This looks more professional without an abbreviation.

```suggestion
Since KDE theming is still a work in progress, some manual steps may be required to properly
apply its theme.
```

(Also, this paragraph may need updating as KDE support has been changed/improved since it was written. No need to address that in this PR though.)